libse/UknownFormatImporter.cs

```diff
@@ -658,6 +660,11 @@ private static Subtitle ImportTimeCodesOnSameSeperateLineNoMilliseconds(string[]
             return subtitle;
         }
 
+        private static string GetLineWithPerhapsOnlyNumbers(string line)
+        {
+            return line.Replace(" ", string.Empty).Replace(".", string.Empty).Replace(",", string.Empty).Replace("\t", string.Empty).Replace(":", string.Empty).Replace(";", string.Empty).Replace("{", string.Empty).Replace("}", string.Empty).Replace("[", string.Empty).Replace("]", string.Empty).Replace("-", string.Empty).Replace(">", string.Empty).Replace("<", string.Empty);
+        }
+
```

Comment:
@niksedk  Is this, perhaps, a **StringExtensions** candidate. Like `Remove<something>` or `Strip...` ?